src/platform-includes/sourcemaps/legacy-troubleshooting/javascript.mdx

```diff
@@ -183,6 +183,12 @@ We maintain an online validation tool that can be used to test your source maps
 
 Alternatively, if you are using Sentry CLI to upload source maps to Sentry, you can use the `--validate` command line option to verify your source maps are correct.
 
+## Verify You Are Running A Production Build
```

Comment:
Should this info go in src/platform-includes/sourcemaps/troubleshooting/javascript.mdx too? so it shows up on /platforms/javascript/sourcemaps/troubleshooting_js/legacy-uploading-methods/ ?

Reply:
Great catch! I knew I was forgetting one location...